jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaConfigPanel.kt

```diff
@@ -97,6 +97,18 @@ class LambdaConfigPanel(private val project: Project) : JPanel(BorderLayout()) {
         iamRole.selected()
             ?: return iamRole.validationInfo(message("lambda.upload_validation.iam_role"))
 
-        return timeoutSlider.validate() ?: memorySlider.validate()
```

Comment:
Just to make sure, the slider increments in 64 increments already?

Reply:
On unselecting the sldier it will snap to 64 increment